recipes/ua-nodeset/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "1.02":
```

I think you need to be more specific with the version.
There are multiple releases of 1.02.  See https://github.com/OPCFoundation/UA-Nodeset/releases
```suggestion
  "1.02-2021-07-21":
```